examples/chatbot/components/conversation.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import Message from './message'
+
+export default React.createClass({
+
+  getDefaultProps() {
+    return {
+      messages: []
+    }
+  },
+
+  componentDidUpdate() {
+    let el = React.findDOMNode(this)
+
+    el.scrollTop = el.scrollHeight
+  },
+
+  getMessage(message) {
+    return (<Message key={ message.id } { ...message } />)
+  },
+
+  render() {
+    const { messages } = this.props
+
+    return (
+      <ol className="conversation">
+        { messages.map(this.getMessage) }
+      </ol>
+    )
+  }
+
```

Comment:
This might be overkill, but it would be fun to add a little a11y with an aria announcement for chat. Meh example: https://github.com/vigetlabs/know-lupus-game/blob/master/front-end/src/assets/js/components/Banner/Announcements.js